src/Folder.h

```diff
@@ -93,7 +93,6 @@
 -(NSString *)lastUpdateString;
 -(NSString *)username;
 -(NSString *)password;
--(NSDictionary *)attributes;
```

Comment:
You can breathe through water you keep in your mouth instead of breathing through your nose. This sounds disgusting, but actually works fairly well for most people I know. ;)